modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/service/DLOpenerOneDriveDLAppServiceWrapper.java

```diff
@@ -171,8 +171,8 @@ public FileEntry updateFileEntryAndCheckIn(
 
 			return super.updateFileEntryAndCheckIn(
 				fileEntryId, sourceFileName, mimeType, title, urlTitle,
-				description, changeLog, dlVersionNumberIncrease, null,
-				expirationDate, revisionDate, serviceContext);
+				description, changeLog, dlVersionNumberIncrease, inputStream,
```

Comment:
can you make the same on line 181
`return super.updateFileEntry(`

Thanks!

Reply:
@AliciaGarciaGarcia updated. How about this implementation : 
if (_dlOpenerOneDriveManager.isConfigured(fileEntry.getCompanyId()) ||
			_dlOpenerOneDriveManager.isOneDriveFile(fileEntry)) {
			checkInFileEntry(
			fileEntryId, dlVersionNumberIncrease, changeLog, serviceContext);
		}		

		return super.updateFileEntry(
			fileEntryId, sourceFileName, mimeType, title, urlTitle, description,
			changeLog, dlVersionNumberIncrease, inputStream, size, expirationDate,
			revisionDate, serviceContext);

Will be shorter but a little bit of confuse.

Reply:
I prefer the previous one. But up to you

Reply:
We could keep it as it is. Thanks.